src/components/forms/formComponents.js

```diff
@@ -171,61 +172,57 @@ export const FormInputMultiText = (config) => {
-        placeholder: placeholder || title,
-        style: { ...styles.inputStyle, ...inputStyle },
-        disabled,
-        'aria-describedby': ariaDescribedby,
-        onKeyUp: (event) => event.code === 'Enter' ? pushValue(event.target) : setValidation({ valid: true }),
-        onFocus: () => setValidation({ valid: true })
-      }),
-      h(button, {
-        className: 'form-btn btn-xs',
-        type: 'button',
-        disabled,
-        style: {
+  return <div>
+    <div className="formControl-group flex-row">
+      <input
+        id={id}
+        name={name || id}
+        type="text"
+        className={`form-control ${!isValid(validation) || !isValid(inputValidation) ? 'errored' : ''}`}
+        placeholder={placeholder || title}
+        style={{ ...styles.inputStyle, ...inputStyle }}
+        disabled={disabled}
+        aria-describedby={ariaDescribedby}
+        onKeyUp={(event) => event.code === 'Enter' ? pushValue(event.target) : setValidation({ valid: true })}
+        onFocus={() => setValidation({ valid: true })}
+      />
+      <button
+        className="form-btn btn-xs"
+        type="button"
+        disabled={disabled}
+        style={{
           marginTop: 0,
           minWidth: 'fit-content'
-        },
-        onClick: () => pushValue(document.getElementById(id))
-      }, [
-        span({
-          className: 'glyphicon glyphicon-plus',
-          'aria-label': 'Add',
-          style: { margin: '0 8px' },
-          isRendered: !disabled
-        })
-      ])
-    ]),
-    errorMessages(inputValidation),
-    errorMessages(validation),
-    div({ className: 'flex-row', style: { justifyContent: 'flex-start' } },
-      formValue.map((val, i) => {
-        return h(button, {
-          key: val,
-          className: 'pill btn-xs',
-          type: 'button',
-          disabled,
-          onClick: () => removePill(i)
-        }, [
-          val,
-          span({
-            className: 'glyphicon glyphicon-remove',
-            style: { marginLeft: '8px' },
-            isRendered: !disabled
-          })
-        ]);
-      })
-    )
-  ]);
+        }}
+        onClick={() => pushValue(document.getElementById(id))}
+      >
+       !disabled && <span
+          className="glyphicon glyphicon-plus"
+          aria-label="Add"
+          style={{ margin: '0 8px' }}
+        />
+      </button>
+    </div>
+    {errorMessages(inputValidation)}
+    {errorMessages(validation)}
+    <div className="flex-row" style={{ justifyContent: 'flex-start' }}>
+      {formValue.map((val, i) => (
+        <button
+          key={val}
+          className="pill btn-xs"
+          type="button"
+          disabled={disabled}
+          onClick={() => removePill(i)}
+        >
+          {val}
+          !disabled && <span
+            className="glyphicon glyphicon-remove"
+            style={{ marginLeft: '8px' }}
+          />
```

Comment:
```suggestion
          {!disabled && <span
            className="glyphicon glyphicon-remove"
            style={{ marginLeft: '8px' }}
          />}
```

Does this need curly braces?